### ISSUE TITLE
HOW to get information about config in class Model(DQNModel)?

### ISSUE THREAD

**Issue by silentobservers** [tensorpack/tensorpack#894]
Hi, nice to meet you.
I wonder how to get information about config, such as  current learning rate or exploration probability,  in class Model(DQNModel), when the program is training. I have tried  some method but failed.

**Comment by ppwwyyxx** [comment]
learning rate is a variable defined in `optimizer()`. You can define it in `build_graph()` instead and access it in `optimizer()`.

The current exploration prob is not a tensorflow variable so there is no way to access it in the graph. You need to modify many code to do that.
